src/modm/platform/adc/at90_tiny_mega/adc_mega.hpp.in

```diff
@@ -283,13 +291,32 @@ public:
 		return (ADCSRA & (1<<ADIF));
 	}
 
+	/// @return	`InterruptFlag::All` if the flag is set,
+	///			`0` otherwise
+	static inline InterruptFlag_t
+	getInterruptFlags()
+	{
+		if (ADCSRA & (1<<ADIF)) {
+			return InterruptFlag::All;
+		} else {
+			return InterruptFlag(0);
+		}
```

Comment:
```suggestion
		return InterruptFlag(ADCSRA & (1<<ADIF));
```
this is what I meant

Reply:
ah. I forgot about this... It's fixed now

Reply:
No problem, I'll test this next week together with the release testing, since I currently don't have access to my hardware box.